spec/pagseguro/payment_request/request_serializer_spec.rb

```diff
@@ -61,125 +22,230 @@
           }
         }
       ]
+      expect(subject.to_xml_params).to match %r[<receivers>.*<receiver>.*<email>a@example.com<\/email>]
```

Comment:
Test is broken, try `expect(subject.to_xml_params).to match %r[<receivers>.*<receiver>.*<email>a@example.com<\/email>]m`